Modules/Devices/src/Devices.Application/Clients/Commands/CreateClients/Handler.cs

```diff
@@ -22,12 +27,27 @@ public async Task<CreateClientResponse> Handle(CreateClientCommand request, Canc
                 throw new ApplicationException(ApplicationErrors.Devices.ClientIdAlreadyExists());
         }
 
+        if (!string.IsNullOrEmpty(request.TierId))
```

Comment:
As discussed in chat: the DefaultTier property in the request should never be empty. **Add a validator to validate that.**